felles/abac/src/main/java/no/nav/vedtak/sikkerhet/populasjon/PopulasjonInternRequest.java

```diff
@@ -0,0 +1,13 @@
+package no.nav.vedtak.sikkerhet.populasjon;
+
+import java.util.Set;
+import java.util.UUID;
+
+import no.nav.vedtak.sikkerhet.kontekst.AnsattGruppe;
+
+public record PopulasjonInternRequest(UUID ansattOid,
+                                      Set<AnsattGruppe> kreverGrupper,
```

Comment:
Hva er det fptilgang skal bruke kreverGrupper til?

Reply:
Sjekke om oid er med i grupper. Tilgang har Entra-grensesnitt
Fiirst cut er at naiserator bare inneholder veileder/saksbehandler/drift - ikke de øvrige.

Reply:
Vil nå potensielt gjøre to kall til FPtilgang
- ett for å sjekke om gruppe-krav fra Fagtilgang er møtt
- ett for populasjonstilgang